📖 [bento] Add documentation for social-share, date-countdown, accordion: 1.0 (#31920)
ampproject/amphtml: extensions/amp-date-countdown/amp-date-countdown.md

| {h} {hours} {m} {minutes}                       | 240 hours 10 minutes                 | `biggest-unit='hours'` |
| {d} {days} {h} {hours} {m} {minutes}            | 50 天 5 小时 10 分钟                 | `locale='zh-cn'`       |

### Migrating from 0.1

> Unlike `0.1`, the experimental `1.0` version of `amp-date-countdown` does not currently support the `data-count-up` attribute which enables the component to count in the opposite direction.

Reviewer: Is this a TODO on tracker?

Author: Yes, this tracked in the tracker here: https://github.com/ampproject/amphtml/issues/30052

> Unlike `0.1`, the experimental `1.0` version of `amp-date-countdown` does not currently support the `data-count-up` attribute which enables the component to count in the opposite direction.

Reviewer 2: ```suggestion
The experimental `1.0` version of `amp-date-countdown` does not support the `data-count-up` attribute. This means the `1.0` component is unable to count in the opposite direction.
```

Author: Thanks, I used this suggestion in the latest revision.

> 

Author: This feature has been implemented in `0.1` only and is fairly new.
